src/Controller/BatchProcessingController.py

```diff
@@ -703,3 +708,46 @@ def create_timestamp(cls):
             str(min) + str(sec)
 
         return time_stamp
+
+    def set_kaplanmeier_targetCol(self, target):
+        self.kaplanmeier_targetCol = target
+
+    def set_kaplanmeier_DurationOfLifeCol(self, duration):
+        self.kaplanmeier_DurationOfLifeCol = duration
+
+    def set_kaplanmeier_AliveOrDeadCol(self, AliveOrDead):
+        self.kaplanmeier_AliveOrDeadCol = AliveOrDead
```

Comment:
pretty sure this against python convention as we can directly access?

Reply:
ah, actually I see that this file uses setters throughout. It would be better then to follow what was previously done as you have :)